geoips/schema/product_defaults/interpolator_algorithm_colormap.yaml

```diff
@@ -1,7 +1,6 @@
 $id: product_defaults.interpolator_algorithm_colormap
 $ref: bases.top
 type: object
-unevaluatedProperties: false
```

Comment:
@jsolbrig Was there a reason "unevaluatedProperties" are now allowed on interpolator_algorithm_colormap products?  Was that intentional?

Reply:
Yes, this was intentional. This was required in order to allow `source_name` and `variables`. Looking at it again, though, I think it should be there and `source_name` and `variables` should be added as possible parameters.

Reply:
I submitted #210 to address this.